### PR TITLE
migrate PIN scrambling to ExtSettings

### PR DESCRIPTION
**Pull request by x86pup**

### CODE REVIEW

File: res/values/strings_ext.xml [hunk 1]
```diff
@@ -14,4 +14,7 @@
     <string name="screenshot_timestamp_exif_title">Save screenshot timestamp to EXIF</string>
     <string name="screenshot_timestamp_exif_summary">Enables adding a timestamp to screenshot EXIF metadata</string>
 
+    <string name="scramble_pin_title">PIN scrambling</string>
+    <string name="scramble_pin_summary">Controls PIN scrambling option when inputting PIN on screen lock.</string>
```
Comment (muhomorr) [inline res/values/strings_ext.xml:18]: Title and summary aren't worded right, I think

Comment (x86pup) [reply res/values/strings_ext.xml:18]: @muhomorr I copy-pasted from the original values, how should they be worded?

Comment (muhomorr) [reply res/values/strings_ext.xml:18]: Hard to come up with a good one. It's a minor thing, can leave it as is.

**Comment by x86pup** [comment]
Merged in https://github.com/GrapheneOS/platform_packages_apps_Settings/pull/152

**Pull request closed without merging**